src/index.ts

```diff
@@ -396,99 +384,99 @@ class ColorPickerWidget extends WidgetType {
-          ) {
-            return false;
-          }
+    }
+  },
+  {
+    decorations: (v) => v.decorations,
+    eventHandlers: {
+      change: (e, view) => {
+        const target = e.target as HTMLInputElement;
+        if (
+          target.nodeName !== 'INPUT' ||
+          !target.parentElement ||
+          !target.parentElement.classList.contains(wrapperClassName)
+        ) {
+          return false;
+        }
 
-          const data = pickerState.get(target)!;
-
-          let converted = target.value + data.alpha;
-          if (data.colorType === ColorType.rgb) {
-            converted = `rgb(${hexToRGBComponents(target.value).join(', ')}${
-              data.alpha
-            })`;
-          } else if (data.colorType === ColorType.named) {
-            // If the hex is an exact match for another named color, prefer retaining name
-            for (const [key, value] of namedColors.entries()) {
-              if (value === target.value) converted = key;
-            }
-          } else if (data.colorType === ColorType.hsl) {
-            const [r, g, b] = hexToRGBComponents(target.value);
-            const [h, s, l] = rgbToHSL(r, g, b);
-            converted = `hsl(${h}, ${Math.round(s * 100)}%, ${Math.round(
-              l * 100,
-            )}%${data.alpha})`;
+        const data = pickerState.get(target)!;
+
+        let converted = target.value + data.alpha;
+        if (data.colorType === ColorType.rgb) {
+          converted = `rgb(${hexToRGBComponents(target.value).join(', ')}${
+            data.alpha
+          })`;
+        } else if (data.colorType === ColorType.named) {
+          // If the hex is an exact match for another named color, prefer retaining name
+          for (const [key, value] of namedColors.entries()) {
+            if (value === target.value) converted = key;
           }
+        } else if (data.colorType === ColorType.hsl) {
+          const [r, g, b] = hexToRGBComponents(target.value);
+          const [h, s, l] = rgbToHSL(r, g, b);
+          converted = `hsl(${h}, ${Math.round(s * 100)}%, ${Math.round(
+            l * 100,
+          )}%${data.alpha})`;
+        }
 
-          view.dispatch({
-            changes: {
-              from: data.from,
-              to: data.to,
-              insert: converted,
-            },
-          });
+        view.dispatch({
+          changes: {
+            from: data.from,
+            to: data.to,
+            insert: converted,
+          },
+        });
 
-          return true;
-        },
+        return true;
       },
     },
-  );
+  },
+);
 
 export const colorPicker = (options: CSSColorPickerOptions = {}): Extension => [
-  colorPickerViewPlugin(options),
-  colorPickerTheme(options),
+  colorPickerViewPlugin,
+  colorPickerTheme,
+  EditorView.theme({
+    [`.${wrapperClassName}`]: options.style?.wrapper,
+    [`.${wrapperClassName} input[type="color"]`]: options.style?.input,
+  }),
```

Comment:
Why should we accept options? Users can set the theme themselves, we export the classname